.env.example

```diff
@@ -4,6 +4,9 @@ ILBOT_WEBHOOK_PROXY_URL=https://smee.io/your-webhook-proxy-url
 # Github App Id after bot's registration.
 ILBOT_GITHUB_INTEGRATION_ID=your-github-app-id
 
+# Taxonomy URL
```

Comment:
Just to give a context. AWS credentials we use for the prod workers, and secrets.enc has all the required parameters set. Currently we only have one dev machine where we can actually hit the backend endpoints. But i think it's a good idea to add these parameters in the .env, in case we want to spawn more dev machine to hack with the backend endpoints. Let's take it through another PR ?

Reply:
Sounds great!